src/Catalyst.Core.Modules.Ledger.Tests/IntegrationTests/LedgerKvmTests.cs

```diff
@@ -0,0 +1,308 @@
+            
+            // migration call
+            string call2Hex = "fdacd5760000000000000000000000000000000000000000000000000000000000000001";
+
+            Delta delta = new Delta
+            {
+                TimeStamp = Timestamp.FromDateTime(DateTime.UtcNow),
+                ContractEntries =
+                {
+                    EntryUtils.PrepareContractEntry(null, sender, 0, migrationInitHex),
+                    EntryUtils.PrepareContractEntry(null, sender, 0, call1Hex, 1),
+                    EntryUtils.PrepareContractEntry(null, sender, 0, initCodeHex, 2),
+                    EntryUtils.PrepareContractEntry(null, sender, 0, call2Hex, 3),
+                }
+            };
+
+            RunDeltas(delta);
+
+            Account migrationAccount = _stateProvider.GetAccount(contractAddress1);
+            migrationAccount.Balance.Should().Be(0);
+            _stateProvider.GetCode(migrationAccount.CodeHash).Should().Equal(Bytes.FromHexString("608060405260043610610062576000357c0100000000000000000000000000000000000000000000000000000000900463ffffffff1680630900f01014610067578063445df0ac146100aa5780638da5cb5b146100d5578063fdacd5761461012c575b600080fd5b34801561007357600080fd5b506100a8600480360381019080803573ffffffffffffffffffffffffffffffffffffffff169060200190929190505050610159565b005b3480156100b657600080fd5b506100bf610241565b6040518082815260200191505060405180910390f35b3480156100e157600080fd5b506100ea610247565b604051808273ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff16815260200191505060405180910390f35b34801561013857600080fd5b506101576004803603810190808035906020019092919050505061026c565b005b60008060009054906101000a900473ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff163373ffffffffffffffffffffffffffffffffffffffff16141561023d578190508073ffffffffffffffffffffffffffffffffffffffff1663fdacd5766001546040518263ffffffff167c010000000000000000000000000000000000000000000000000000000002815260040180828152602001915050600060405180830381600087803b15801561022457600080fd5b505af1158015610238573d6000803e3d6000fd5b505050505b5050565b60015481565b6000809054906101000a900473ffffffffffffffffffffffffffffffffffffffff1681565b6000809054906101000a900473ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff163373ffffffffffffffffffffffffffffffffffffffff1614156102c957806001819055505b505600a165627a7a723058206bf582fa33b86704b115209928731f910b5f9872d5a4c376fe8d639aa373ad790029"));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Split this 1622 characters long line (which is greater than 200 authorized).](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4350015)